### DISCUSSION TITLE
重复：自定义快捷键

### DISCUSSION THREAD

**Discussion by laulguo** [hooke007/mpv_PlayKit#265] [回收]
如何保存自定义的快捷键呢，目前我是直接扔在了input_uosc.conf内，但是随着版本更新很容易被覆盖掉，看了一下直接写个input.conf好像无效

**Comment by hooke007** [comment]
dup https://github.com/hooke007/MPV_lazy/discussions/117 https://github.com/hooke007/MPV_lazy/discussions/201

每次更新后手动修改
